live-examples/js-examples/date/date-tolocaledatestring.html

```diff
@@ -11,5 +11,10 @@
 
 console.log(event.toLocaleDateString('ko-KR', options));
 // expected output: 2012년 12월 20일 목요일
+
+// Pass undefined as the first argument to use the browser's default locale
```

Comment:
Please remove this line: we don't generally use descriptive comments in the examples unless there's something very unusual to point out.